tests/unit_tests/test_hyde.py

```diff
@@ -4,9 +4,9 @@
 import numpy as np
 from pydantic import BaseModel
 
+from langchain.chains.hyde.base import HypotheticalDocumentEmbedder
```

Comment:
nit: can we move this into the unit_tests/chains folder now that its a chain?

Reply:
done